pkg/cmd/cloudbees/cloudbees.go

```diff
@@ -1,224 +0,0 @@
-package cloudbees
-
-import (
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"os/exec"
-	"os/signal"
-	"syscall"
-	"time"
-
-	"github.com/jenkins-x/jx/pkg/cmd/helper"
-	"github.com/jenkins-x/jx/pkg/log"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-	"gopkg.in/AlecAivazis/survey.v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/jenkins-x/jx/pkg/cmd/opts"
-	"github.com/jenkins-x/jx/pkg/cmd/templates"
-	"github.com/jenkins-x/jx/pkg/util"
-	"github.com/pkg/browser"
-)
-
-// CloudBeesOptions are the options to execute the jx cloudbees command
-type CloudBeesOptions struct {
-	*opts.CommonOptions
-
-	OnlyViewURL  bool
-	HideURLLabel bool
-	LocalPort    string
-}
-
-const (
-	// DefaultForwardPort is the default port that the UI will be forwarded to
-	DefaultForwardPort = "9000"
-)
-
-var (
-	core_long = templates.LongDesc(`
-		Opens the CloudBees JX UI in a browser.
-
-		Which helps you visualise your CI/CD pipelines.
-`)
-	core_example = templates.Examples(`
-		# Open the JX UI dashboard in a browser
-		jx cloudbees
-
-		# Print the Jenkins X console URL but do not open a browser
-		jx console -u`)
-)
-
-func NewCmdCloudBees(commonOpts *opts.CommonOptions) *cobra.Command {
-	options := &CloudBeesOptions{
-		CommonOptions: commonOpts,
-	}
-	cmd := &cobra.Command{
-		Use:     "cloudbees",
-		Short:   "Opens the CloudBees app for Kubernetes for visualising CI/CD and your environments",
-		Long:    core_long,
-		Example: core_example,
-		Aliases: []string{"cloudbee", "cb", "ui", "jxui"},
```

Comment:
On closer inspection this should not be removed ;-)

This is the `jx ui` command which does the port forward to open the UI on localhost in case the UI is not installed with SSO. 

I think it is unfortunate that the commands main name is 'cloudbees'. I guess the description is also a bit off as well. 

Given that the way we document this in the distribution docs is `jx ui`, what we could do is:

* make the main name of this command "ui"
* rename file and package to get rid of 'cloudbees'
* add 'cloudbees' as alias for good measure in case someone is actually using it

BTW,  that seems wrong as well:

> 		# Print the Jenkins X console URL but do not open a browser
>		jx console -u`)

Should probably be `jx ui -u`

Reply:
whoops! I'll clean this up on Monday.